versions.json

```diff
@@ -825,8 +825,8 @@
     },
     "jwt-cpp": {
         "enable_sha": true,
-        "commitish": "ce1f9df3a9f861d136d6f0c93a6f811c364d1d3d",
-        "version_string": "0.6.99.0",
+        "commitish": "6dd190d154db29fad071574aa5558f15a70752ad",
+        "version_string": "0.7.99.0",
```

Comment:
Bumped version string to `0.7.99.0` vs `0.6.99.0`. This is to show that we are ahead of v0.7.0-rc0.

Reply:
@korydraughn suggested that we may also use the first 8 characters of the commitish for the version_string:

https://github.com/irods/externals/blob/49eef15b0f46785a5005c1bb39ecd15eab6cf665/versions.json#L305-L307

Reply:
i like this - doesn't stomp on any possible future real version number

Reply:
I very much prefer version strings to be sortable when possible.

Reply:
This should probably be bumped to 0.6.99.1, as 0.7.0 (non-RC) isn't out yet, so 0.7.anything would put us ahead of a future release.

Reply:
Updated to 0.6.99.1.